astropy/modeling/utils.py

```diff
@@ -484,3 +484,64 @@ def combine_labels(left, right):
 
     return left + right
 
+
+def render_models2sky(model_list, image, x_key='x_mean', y_key='y_mean'):
```

Comment:
I'd lean toward re-naming the Gaussian parameters for consistency among the models. It might be useful to still have x_key and y_key to handle user-defined custom models, but I could go either way on that.